hphp/hack/src/heap/hh_shared.c

```diff
@@ -142,11 +142,11 @@
 #if !defined __APPLE__ && !defined _WIN32
   // Linux version for the architecture must support syscall memfd_create
```

Comment:
@Orvid yes `SYS_memfd_create` is defined on  powerpc64le-linux-gnu/sys/syscall.h but defined as: 

`#define SYS_memfd_create __NR_memfd_create`

What do you suggest in this case? Originally my pull request check `#ifndef __NR_memfd_create` should i change back to it?